code/game/sound.dm

```diff
@@ -184,6 +184,31 @@ distance_multiplier - Can be used to multiply the distance at which the sound is
 			var/mob/M = m
 			M.playsound_local(M, null, volume, vary, frequency, null, channel, pressure_affected, S)
 
+/proc/play_soundtrack_music(sound_path, list/hearers = null, volume = 100, ignore_prefs = FALSE, play_to_lobby = FALSE, allow_deaf = TRUE, specific_z = null)
+	var/sound/S = sound(sound_path, volume=volume, wait=0, channel=CHANNEL_AMBIENT_MUSIC)
+	. = S
+
+	if (hearers == null)
+		hearers = GLOB.player_list
+
+	for(var/mob/M in hearers)
+		if (!ismob(M))
+			continue
```

Comment:
```suggestion
```

Reply:
Makeshift isnull test

Reply:
how can there be null in hearers list? 
just check for client instead

Reply:
Nulls have historically shown up in just about any global list